### DISCUSSION TITLE
Blog Post on Future of Text in WebXR

### DISCUSSION THREAD

**Discussion by jimmy6DOF** [omigroup/media#86] [Ideas]
### The Dawn of a New Era: Spatial Computing Shakes up the Landscape of Text Reading

Imagine stepping into a world where reading is no longer a linear journey, confined to the boundaries of static pages or even screens. This, my friends, is the exciting future that "The Future of Text" foundation recently unveiled at the WebXR monthly convention. A revolutionary union of Virtual Reality (VR), Augmented Reality (AR), and mixed reality (MR) into a single seamless entity - Spatial Computing.

### A Leap from 2D to 3D is Spatial Computing

"Spatial Computing," the buzzword that's been stirring up discussions among tech enthusiasts, extends the physical world we inhabit by integrating digital elements into our environments and transforming the way we see, interact, and understand information. Simply put, it's "living" your data!

The shift from 2D text interfaces to 3D environments marks a significant moment in the evolution of spatial computing. While traditional 2D platforms have served us well, they limit our cognitive potential, merely scratching the surface of our brain's spatial capabilities. Here is where VR steps into the spotlight, providing a conduit for exploring the world of text in 3D interfaces.

But what does Spatial Computing mean for text reading? Enter the next generation Spatial Computing Text Reader. By exploiting extended reality's (XR) immersive nature, this innovative tool redefines the boundaries of text consumption, making it an interactive and engaging experience – literarily a walk in the VR park of words! Feel a childlike wonder as visual and textual elements sway and dance around you, inviting you to dive deeper into the abyss of information and knowledge.

### A Symphony of Perspectives on Spatial Computing

While the excitement is palpable, there's also an array of perspectives surrounding this new avatar of text consumption. Embracing such a radical development comes with its own set of challenges. Will it make reading more accessible, or create a new digital divide? Can it replace the good ol' book or the crisp Kindle screen with a spatial experience?

However, as varied as the views may be, it's clear that we are sitting on the precipice of a digital revolution. Spatial Computing is here to stay, and it's set to disrupt our understanding and interaction with information.

### Striking a Balance with Data

The sheer potential of Spatial Computing is supported by promising data. According to the International Data Corporation, the AR and VR market is projected to reach $162 billion by 2024. Furthermore, with the use of WebXR, a unified framework for VR and AR experiences on the web, the technology's accessibility is greatly enhanced.

Yet, despite these figures, caution and balance are key. After all, technology is only as effective as its practical applicability. Therefore, the development and implementation of Spatial Computing as a viable 'Tool for Thought' must involve comprehensive user experience research and remain sensitive to digital inclusivity.

#### Actionable Insights: Embracing Spatial Computing

For the avid technophiles and XR enthusiasts seeking to plunge into the Spatial Computing world, here are a few tips:
- Stay Updated: Keep an eye out for the exciting innovations and updates The Future of Text foundation will release.
- Learn the Lingo: Familiarize yourself with terms such as 'WebXR', 'Spatial Computing', 'XR', and more.
- Start Small: Don't jump into creating massive immersive worlds; start with encapsulating individual ideas in smaller environments. Simplicity is key.
- Grow Gradually: Once you find your comfort zone, you can start exploring grander scales. Migrate your digital notebooks, Mind Maps, or concept maps into more complex and rich 3D structures.
- Experiment: Utilizing VR headsets or even your smartphone, delve into various WebXR experiences available. There’s nothing like firsthand experience to really appreciate what Spatial Computing has to offer.

### A Brave New World of Reading

We stand at the crossroads of a historical shift where we leave the flat world of text behind and step into the rich multifaceted realm of 3D spatial interfaces. As we envision and build these new Tools for Thought, incorporating text into 3D environments could empower us to understand, remember, and apply complex information like never before, riding the wave of revolution in search, organization, and recall. Let's surf.

As we warp our way into the future of text, we step encumbered yet exhilarated into a new thrilling world of reading – a universe adorned not with walls of text, but with constellations of ideas, narratives and data points. The promise of a more immersive and interactive experience dangles within our grasp.

So throw off the bowlines and explore this ocean of possibilities. Discover how The Future of Text and Spatial Computing plan to revolutionize our reality – one character, one word, one sentence at a time. And remember, in the universe of Spatial Computing, the text is not just alive; you're living it!

TODO:
Scrape Quotes from [Twitch Recording](https://www.twitch.tv/webxrdiscord) 
Proof Read from [FoT Project](https://thefutureoftext.org/) 
Run story building through [this project](https://github.com/hrishioa/lumentis) to produce blog content on the fly during interviews

**Comment by indiebio** [comment]
This is :fire: , good one jimmy! Are you going to add the hyperlinks to the future of text and webxr meetup?

**Comment by jimmy6DOF** [comment]
Managed to get the transcript and Diarize it for speakers +/- level of accuracy

Now will get it onto this tool for auto chop chop blog post and see what we get back to review, redline a little, and pop out. 

[WebXR Monthly Meetup - Future_of_Text.txt](https://github.com/omigroup/media/files/14922466/WebXR.Montly.Meetup.-.Future_of_Text.txt)

**Comment by jimmy6DOF** [comment]
# THE FUTURE OF TEXT IN WEBXR

imagine stepping into a world where reading is no longer a linear journey, confined to the boundaries of static pages or even screens. This is the exciting future that "The Future of Text" foundation recently discussed at the WebXR monthly meetup.   

The shift from 2D text interfaces to 3D environments marks a significant moment in the evolution of spatial computing. While traditional 2D platforms have served us well, they limit our cognitive potential, merely scratching the surface of our brain's spatial capabilities. Here is where VR steps into the spotlight, providing a conduit for exploring the world of text in 3D interfaces.

As Frode says: "You know, there's a lot at stake here ... in the near future, let's say in 10 years, everybody will have a headset in their bag. But whether it's going to be useful to work with like a thinking cap, which is how we think of it now, or if it's going to be for watching movies, playing games, and so on, that's entirely up to people like us."

# Introduction

The Future of Text group is kicking off on an ambitious new project developing answers questions like how will the advent of dynamic embodied 3D spatial computing affect our ability to interact with text and documents in general.  The goal is to develop and experiment how to produce, format, and consume text in  environments like VR and AR (collectively known as "extended realities" or XR) using the open web standard WebXR.  Dene Grigar and Frode Hegland are co-PIs of the initiative, made possible by the Alfred P. Sloan Foundation. 


<Callout>
One key motivation is to deliver immersive spaces without relying on proprietary software on the user side. By embracing open standards, the team aims to foster wide spread innovation and collaboration in the field of digital text and spatial computing.
</Callout>


The project involves three main approaches:

1. **Developing software**: Creating innovative applications and tools that push the boundaries of how text can be manipulated, navigated, and experienced in XR environments.

2. **Metadata as connective tissue**: Exploring how metadata can act as a bridge between different systems, enabling seamless integration and interoperability of text-based experiences across various platforms.  See the section on Visual Meta

3. **Dialogue and community building**: Facilitating discussions, organizing symposiums, and publishing books to bring together researchers, developers, and thought leaders in this emerging field.

<Steps>

### Step 1: Explore the Prototypes

To get a hands-on understanding of the project's goals, you can explore the prototypes that Dene and Frode have developed so far:

- Try out the [word processor](https://www.augmentedtext.info/author) and [PDF viewer](https://www.augmentedtext.info/reader) built natively for the Apple Vision Pro and Quest 3 headsets.

- Experience the [Future Text Lab](https://thefutureoftext.org/), a cutting-edge research environment focused on academic reading and authoring in XR.

#### And to Get Involved .... 

### Step 2: Attend the Annual Symposium

Dene and Frode invite you to attend the [annual Future of Text symposium](https://thefutureoftext.org/symposium/), where researchers, developers, and industry experts gather to discuss the latest advancements and challenges in the field of digital text and spatial computing.

</Steps>

One of the primary challenges is the need to rethink the way we interact with and navigate text in a 3D space. In a 2D environment, we are familiar with scrolling, clicking on hyperlinks, and using menus and toolbars. However, in an XR environment, these traditional interactions may not be as intuitive or effective.

By embracing the potential of XR technologies, the Future of Text project aims to redefine the way we think about and interact with text, paving the way for a future where immersive environments become an integral part of academic work and communication.

* this summary is based on [a presentation given to the WebXR monthly meetup](https://www.twitch.tv/videos/2107352646)
